docs/src/pages/docs/crocks/Async.md

```diff
@@ -1257,6 +1257,69 @@ timeout(slow)
 
 ## Transformation Functions
 
+#### asyncToPromise
+
+`crocks/Async/asyncToPromise`
+
+```haskell
+asyncToPromise :: Async e a -> Promise a e
```

Comment:
Need to provided the dual signature here. like the examples below this function

Reply:
added the new sig